tensorboard/program.py

```diff
@@ -118,14 +118,17 @@ def __init__(self,
     """Creates new instance.
 
     Args:
-      plugins: A list of TensorBoard plugins to load, as TBLoader instances or
-        TBPlugin classes. If not specified, defaults to first-party plugins.
+      plugin: A list of TensorBoard plugins to load, as TBPlugin classes or
+        TBLoader instances or classes. If not specified, defaults to first-party
+        plugins.
       assets_zip_provider: Delegates to TBContext or uses default if None.
       server_class: An optional factory for a `TensorBoardServer` to use
         for serving the TensorBoard WSGI app. If provided, its callable
         signature should match that of `TensorBoardServer.__init__`.
 
-    :type plugins: list[Union[base_plugin.TBLoader, Type[base_plugin.TBPlugin]]]
+    :type plugins: list[Union[base_plugin.TBLoader,
+                              Type[base_plugin.TBLoader],
```

Comment:
I just got rid of these `:type` lines, which was my original instinct since they're a pain to format legibly and we don't use them consistently or in an enforceable way.